### DISCUSSION TITLE
sqitch tag multiple sqitch projects

### DISCUSSION THREAD

**Discussion by joaostorrer** [theory/sqitch#754] [General]
I've been using sqitch for a while and I have scenarios that to deploy a single database project I create multiples sqitch projects (each one in a different directory).
This is necessary so the database project can be configurable between different clients (one client will have feature A and feature B, and other client will have only feature B for example).

So the folder structure is like this:

Project
├── FeatureA
└── FeatureB

After the development process is done I need to "sqitch tag" each one of the sqitch projects.
It would be nice if I could call "sqitch tag" in the root directory (folder named Project) and the sqitch projects that exists inside this directory were tagged with the same version and note.

**Comment by theory** [comment]
Unfortunately in this case, the only way you could do that is if each project shared the same plan file. Which I suspect, since each project has different changes, it couldn't not. The tag is inserted into the plan file, not a configuration file. The best way to simplify it would be to create some sort of function that finds all the plans and tags them, something like:

``` sh
for plan in $(find . -name '*.plan'); do
    sqitch --plan-file "$plan" tag xyz -m 'Tag xyz'
done
```

**Comment by joaostorrer** [comment]
sqitch --plan-file tag seems to be not working anymore. I'm using sqitch version v1.3.1.
I've tried to call the command different ways  but the result is always the same.
Here is the trace:

$ sqitch --plan-file Age/sqitch.plan tag v0.1 -m 'v0.1' -VVV

No project configuration found. Run the "init" command to initialize a project
trace: Trace begun at C:\Strawberry\perl\site\lib\App\Sqitch\Target.pm line 253
trace: App::Sqitch::Target::BUILDARGS('App::Sqitch::Target', 'sqitch', 'App::Sqitch=HASH(0x4d582c4)', 'plan_file', 'Path::Class::File=HASH(0x4d4990c)', 'name', undef) called at (eval 336) line 34
trace: App::Sqitch::Target::new('App::Sqitch::Target', 'sqitch', 'App::Sqitch=HASH(0x4d582c4)', 'plan_file', 'Path::Class::File=HASH(0x4d4990c)', 'name', undef) called at C:\Strawberry\perl\site\lib\App\Sqitch\Command.pm line 227
trace: App::Sqitch::Command::try {...}  at C:\Strawberry\perl\lib\Try\Tiny.pm line 100
trace: eval {...} at C:\Strawberry\perl\lib\Try\Tiny.pm line 93
trace: Try::Tiny::try('CODE(0x4c73954)', 'Try::Tiny::Catch=REF(0x4d57e5c)') called at C:\Strawberry\perl\site\lib\App\Sqitch\Command.pm line 232
trace: App::Sqitch::Command::parse_args('App::Sqitch::Command::tag=HASH(0x4d57dcc)', 'names', 'ARRAY(0x4d58f0c)', 'all', 0, 'args', 'ARRAY(0x4d5c074)', 'no_changes', 1) called at C:\Strawberry\perl\site\lib\App\Sqitch\Command\tag.pm line 58
trace: App::Sqitch::Command::tag::execute(undef, 'v0.1') called at C:\Strawberry\perl\site\lib\App\Sqitch.pm line 207
trace: App::Sqitch::try {...}  at C:\Strawberry\perl\lib\Try\Tiny.pm line 100
trace: eval {...} at C:\Strawberry\perl\lib\Try\Tiny.pm line 93
trace: Try::Tiny::try('CODE(0x371b4dc)', 'Try::Tiny::Catch=REF(0x4c66784)') called at C:\Strawberry\perl\site\lib\App\Sqitch.pm line 227
trace: App::Sqitch::go('App::Sqitch') called at C:\Strawberry\perl\site\bin\sqitch line 15
